march_joint_inertia_controller/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>march_joint_inertia_controller</name>
+  <version>0.0.0</version>
+  <description>The march_joint_inertia_controller package</description>
+
+  <maintainer email="cilia@todo.todo">cilia</maintainer>
+
+  <license>TODO</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>controller_interface</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_export_depend>controller_interface</build_export_depend>
+  <build_export_depend>pluginlib</build_export_depend>
+  <exec_depend>controller_interface</exec_depend>
+  <exec_depend>pluginlib</exec_depend>
```

Comment:
When dependencies should be used for build, export and exec you can shorten the tags to `<depend>` (see https://wiki.ros.org/Manifest#catkin.2Fpackage.xml.Dependencies). Futhermore, you must add `roscpp`, since that is also listed in your `CMakeLists.txt` and it is necessary for cpp projects.
```suggestion
  <depend>roscpp</depend>
  <depend>controller_interface</depend>
  <depend>pluginlib</depend>
  <depend>controller_interface</depend>
```